pkg/common/diskutil/file_test.go

```diff
@@ -0,0 +1,55 @@
+package diskutil
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test(t *testing.T) {
+	dir, err := ioutil.TempDir("", "test")
+	require.NoError(t, err)
+	defer os.RemoveAll(dir)
+
+	tests := []struct {
+		name string
+		data []byte
+		mode os.FileMode
+	}{
+		{
+			name: "basic test",
+			data: []byte("Hello, World"),
+			mode: 0600,
+		},
+		{
+			name: "empty",
+			data: []byte{},
+			mode: 0440,
+		},
+		{
+			name: "binary",
+			data: []byte{0xFF, 0, 0xFF, 0x3D, 0xD8, 0xA9, 0xDC, 0xF0, 0x9F, 0x92, 0xA9},
+			mode: 0644,
+		},
+	}
+	for _, tt := range tests {
```

Comment:
The loop variable (`tt`) needs be be aliases or the linter is going to complain (it happened to pass CI because there is a bug in golangci-lint that has yet to be fixed that causes it to not run occasionally)

Reply:
ack.  For future reference:

```
spire/pkg/common/diskutil test-diskutils golangci-lint run
file_test.go:41:33: Using the variable on range scope `tt` in function literal (scopelint)
                        err := AtomicWriteFile(file, tt.data, tt.mode)
                                                     ^
file_test.go:46:27: Using the variable on range scope `tt` in function literal (scopelint)
                        require.EqualValues(t, tt.mode, info.Mode())
                                               ^
file_test.go:50:21: Using the variable on range scope `tt` in function literal (scopelint)
                        require.Equal(t, tt.data, content)
                                         ^
```